tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,7 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
```

Comment:
It looks like no asserts were added. Please be careful to make sure you don't remove the Exception until you've added asserts so you remember where you still need to make changes.